### DISCUSSION TITLE
it/trentino-trasporti-digitale/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#2] [Comments]
# it/trentino-trasporti-digitale/

Trentino Trasporti gestisce il trasporto pubblico in Trentino, prova ad essere moderna ma fallisce miseramente. Preparate i popcorn.

https://blog.contrini.it/it/trentino-trasporti-digitale/

**Comment by fcucino** [comment]
Mai provato ad aprire le pagine nel sottomenù [`TARIFFE`](https://www.trentinotrasporti.it/tariffe/abbonamenti/abbonamento-lavoratori) del sito di Trentino Trasporti da cellulare?
Metà del testo fuori dallo schermo e la pagina non è scrollabile... 😞

**Comment by pesaventofilippo** [comment]
Applausi. Sarebbe da stampare e inviare via fax agli uffici di Trentino Trasporti e/o OpenMove. Almeno lo leggono (forse, se va il fax)

**Comment by Samu01Tech** [comment]
Capolavoro di articolo, ho spulciato per caso la web app dell'Alto Adige e si piange anche lì dalle risate. Ho al momento la beta di Open Move ed essendo uno dei primi che ha provato la validazione con bluetooth devo dire che secondo me è il metodo migliore.

**Reply by matteocontrini** [reply]
È una buona idea, sì, speriamo. Ma se l'app non agevola il processo di validazione rischia comunque di essere inutilmente macchinoso. Ad esempio l'app potrebbe ricordarsi l'ultimo metodo di validazione, non ha molto senso doverlo selezionare a mano ogni volta. Oppure potrebbe permettere di aggiungere una shortcut diretta alla home (su Android almeno). Sono cose semplicissime, eppure...

**Reply by pesaventofilippo** [reply]
Esatto, ma in ogni caso direi che il problema maggiore è che se non funziona l'app, anche dopo che hai validato il biglietto, non c'è modo di dimostrarlo. L'app può anche essere magica e validare il biglietto mentre sali sull'autobus, ma se poi non puoi dimostrarlo e ti trattano come colpevole fino a prova contraria...

**Comment by Praisethefab** [comment]
Bell'articolo, aggiungo una cosa bellissima di OpenMove, quando fai un account non controllano che la mail inserita esista.
Quindi non fanno neanche il controllo che tu abbia accesso alla mail quando fai un account, è successo a me quando ho fatto l'account scrivendo @unitn.it invece di @studenti.unitn.it, quando mi sono accorto del mio errore (ho dovuto chiamare il call center che per fortuna è stato abbastanza veloce a rispondere e che mi ha fatto notare questa cosa quando stavo provando a fare il login con @studenti.unitn.it che ovviamente non funzionava) oramai avevo già associato l'abbonamento.
